dqc_us_rules/dqc_us_0015.py

```diff
@@ -72,7 +73,7 @@ def filter_negative_number_facts(val, blacklist_concepts):
     # other filters before running negative numbers check
     # numeric_facts has already checked if fact.value can be made into a number
     facts_to_check = [
-        f for f in numeric_facts if float(f.value) < 0 and
+        f for f in numeric_facts if f.xValid >= VALID and f.xValue < 0 and
```

Comment:
Just looked it up the is VALID call and apparently it is an int and values >= to it are valid, but it is still confusing. I agree with Andrew.